sample-apps/02-otel-collector/kubernetes/01-configmap.yaml

```diff
@@ -5,20 +5,55 @@ metadata:
   namespace: otel-collector
 data:
   otel-collector-config.yml: |
+    extensions:
+      sigv4auth:
+        region: "__replace-region__"
+        service: "osis"
+
     receivers:
       otlp:
         protocols:
           grpc:
             endpoint: 0.0.0.0:55680
+          http:
+            endpoint: 0.0.0.0:55681
             
+    processors:
+      memory_limiter:
+        check_interval: 1s
+        limit_percentage: 80
+        spike_limit_percentage: 30
+      batch:
+        send_batch_size: 10
+        timeout: 5s
+    
     exporters:
-      otlp/data-prepper:
-        endpoint: data-prepper.data-prepper.svc.cluster.local:21890
-        tls:
-          insecure: true
+      otlphttp/traces:
+        traces_endpoint: "https://__REPLACE WITH OtelTraces ADDRESS__/v1/traces"
```

Comment:
Change to:
`traces_endpoint: "https://__REPLACE_WITH_OtelTraces_ENDPOINT__/v1/traces"`